source/models/row.py

```diff
@@ -14,32 +14,29 @@ def get_primary_key(self):
 
     def serialize(self):
         self.__lock()
-        self.__colomns_name_validate()
+        self.colomns_name_validate(self.get_value())
```

Comment:
The Row object shouldn't have been created if the value is wrong.
You need to call this method in the constructor.
`Row.colomns_name_validate(value)`. <-- value here not self.value

Reply:
i want self to take table metadata

Reply:
but i prefered to be instance method